.github/workflows/compile-docker.yml

```diff
@@ -48,6 +48,8 @@ jobs:
           push: true
           labels: ${{ steps.meta.outputs.labels }}
           tags: ${{ steps.meta.outputs.tags }}
+          cache-from: type=registry,ref=nebraltd/hm-diag:buildcache
```

Comment:
What work still needs to be done [here](https://github.com/NebraLtd/product-management/issues/53) given this change?

Reply:
Similar change will have to be done for other repositories.